packages/mdc-select/mdc-select.scss

```diff
@@ -177,3 +177,66 @@
     }
   }
 }
+
+/* postcss-bem-linter: define multi-select */
```

Comment:
supernit: we're trying to switch to scss comments for everything, rather than css comments